controllers/openshift_controller.go

```diff
@@ -300,34 +168,33 @@ WantedBy=multi-user.target
 
 	if kataOC {
 		machinePool = "kata-oc"
-	} else if _, ok := r.kataConfig.Spec.KataConfigPoolSelector.MatchLabels["node-role.kubernetes.io/"+machinePool]; ok {
-		r.Log.Info("in newMCforCR machinePool" + machinePool)
-	} else {
+	} else if _, ok := r.kataConfig.Spec.KataConfigPoolSelector.MatchLabels["node-role.kubernetes.io/"+machinePool]; !ok {
 		r.Log.Error(err, "no valid role for mc found")
 	}
 
 	file := ignTypes.File{}
-	c := ignTypes.FileContents{}
+	c := file.Contents
 
 	dropinConf, err := generateDropinConfig(r.kataConfig.Status.RuntimeClass)
```

Comment:
Do we still need the drop-in config given the config will come from kata rpm ?

Reply:
no, but the rpm is not yet in a nightly build that passes the tests. So we can't test it in combination with the extensions. I would suggest we merge this first and when we can test the new RPM we will merge Fabianos PRs #80 and PR #81 which remove all of the dropin file related code